optimizely/event/payload.py

```diff
@@ -61,10 +61,20 @@ def get_event_params(self):
 class Decision(object):
     """ Class respresenting Decision. """
 
-    def __init__(self, campaign_id, experiment_id, variation_id):
+    def __init__(self, campaign_id, experiment_id, variation_id, metadata):
         self.campaign_id = campaign_id
         self.experiment_id = experiment_id
         self.variation_id = variation_id
+        self.metadata = metadata
+
+
+class Metadata(object):
```

Comment:
nit. Would recommend making this a more descriptive name like `EventMetadata`

Reply:
it is metadata in TDD and go-sdk implementation, so I just followed that

Reply:
I think it is too generic.

Reply:
if we are sending an event, I don't think there is a need to have EventMetadata, it already implies that the metadata belongs to an event.